packages/firebase_auth/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.14.0+2
+
+* Update example app to show token custom claims
+* Rely on custom claims to get the field from the IdTokenResult
```

Comment:
The CHANGELOG entry is kind of vague; it would be helpful to be as specific as possible about user-facing differences in behavior that might be expected from this change.